Site/run.php

```diff
@@ -8,7 +8,7 @@
  * @copyright Ashley Kitson, 2016, UK
  * @license GPL V3+ See LICENSE.md
  */
-require '../vendor/autoload.php';
+require __DIR__.'/../vendor/autoload.php';
```

Comment:
surprised that you need this. What OS is it running on?  In any case, better to use realpath(__DIR__ . '/..') . '/vendor/autoload.php' or dirname(__DIR__) ... in the event that some OSs have a problem.